org.sil.scripture-app-builder.yml

```diff
@@ -0,0 +1,160 @@
+      - name: libXaw
+        sources: &libXaw-sources
+          - type: archive
+            url: https://www.x.org/archive/individual/lib/libXaw-1.0.16.tar.xz
+            sha256: 731d572b54c708f81e197a6afa8016918e2e06dfd3025e066ca642a5b8c39c8f
+      - name: ncurses
+        sources:
+          - type: archive
+            url: https://invisible-mirror.net/archives/ncurses/ncurses-6.4.tar.gz
+            sha256: 6931283d9ac87c5073f30b6290c4c75f21632bb4fc3603ac8100812bed248159
+
+  - name: python3-aeneas
+    buildsystem: simple
+    build-commands:
+      - python3 -m pip install --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "numpy" --no-build-isolation
+      - python3 -m pip install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "aeneas" --no-build-isolation
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/e5/92/88d421001bb257588df4864ceca24d570e2e822db4f40f48737a78b648b2/aeneas-1.7.3.0.tar.gz
+        sha256: b492d8ba36f12cce69c29c35e7075857a9d036c4743cc8f4689922be980826da
+      - type: file
+        url: https://files.pythonhosted.org/packages/4c/f3/038b302fdfbe3be7da016777069f26ceefe11a681055ea1f7817546508e3/soupsieve-2.5-py3-none-any.whl
+        sha256: eaa337ff55a1579b6549dc679565eac1e3d000563bcb1c8ab0d0fefbc0c2cdc7
+      - type: file
+        url: https://files.pythonhosted.org/packages/63/f7/ffbb6d2eb67b80a45b8a0834baa5557a14a5ffce0979439e7cd7f0c4055b/lxml-5.2.2.tar.gz
+        sha256: bb2dc4898180bea79863d5487e5f9c7c34297414bad54bcd0f0852aee9cfdb87
+      - type: file
+        url: https://files.pythonhosted.org/packages/a0/41/8f53eff8e969dd8576ddfb45e7ed315407d27c7518ae49418be8ed532b07/numpy-1.25.2.tar.gz
+        sha256: fd608e19c8d7c55021dffd43bfe5492fab8cc105cc8986f813f8c3c048b38760
+      - type: file
+        url: https://files.pythonhosted.org/packages/b1/fe/e8c672695b37eecc5cbf43e1d0638d88d66ba3a44c4d321c796f4e59167f/beautifulsoup4-4.12.3-py3-none-any.whl
+        sha256: b80878c9f40111313e55da8ba20bdba06d8fa3969fc68304167741bbf9e082ed
+    modules:
+      - name: espeak
+        no-autogen: true
+        subdir: src
+        cleanup:
+          - "*.a"
+        sources:
+          - type: git
+            url: https://salsa.debian.org/a11y-team/espeak.git
+            commit: 0dd86a59e84a475bdd809a3aec7bc95a460ad5a6
+          - type: patch
+            path: patches/espeak/FlatpakEspeak.patch
+
+  - name: openjdk17
+    buildsystem: simple
+    build-commands:
+      # jre install script from extensions location
+      - "/usr/lib/sdk/openjdk17/install.sh"
+      # jdk
+      - "/usr/lib/sdk/openjdk17/installjdk.sh"
+      # exports
+      - "/usr/lib/sdk/openjdk17/enable.sh"
+      # Rename
+      - mkdir /app/jdk17.0.11
+      - mv /app/jdk/* /app/jdk17.0.11/
+      - rm -r /app/jdk
+    finish-args:
+      - "--env=PATH=/app/jre/bin:/usr/bin:$PATH"
```

Comment:
```suggestion
```

This does not work here. Also variable substitution does not work in `finis-args`

You need to pass the entire thing: `--env=PATH=/app/jre/bin:/app/bin:/usr/bin`

Reply:
I have cleaned up openjdk installation.